logo: Make the logo 75% lighter

ochosi opened the pull request:
Previously, the jpg was roughly 4000*1000px, I have scaled this down to half the width now. 2000px felt ok.
Also, I dropped the quality/compression level from 92 to 90, further reducing the size. So hopefully this loads fairly instantaneously now.

Fixes #12

ondrejbudai commented:
Hmm, now I wonder if the formatting is alright with all those indented bullet points. Have you tried building the posts, @dvdhrm?

larskarlitski commented:
> Hmm, now I wonder if the formatting is alright with all those indented bullet points. Have you tried building the posts, @dvdhrm?

Yeah, that will look weird. I've always moved the bullets to the left and reformatted the paragraphs.

dvdhrm commented:
It matches the formatting we have in `NEWS.md`. Both are markdown files.

I don't mind using something else on the website, but maybe we can use the same formatting for both? So far, I liked that `NEWS.md` ended up being formatted as `blockquote`, and thus it looks more like a classic changelog, than like prose. But I am also fine with removing the indentation.

Note that the formatting of the PR as it is looks perfectly fine. It basically emits it as `<pre>`, and since the original is already limited to 80ch, it looks fine. But it is kinda imcompatible with screen-readers and other stuff, which will trip over.

Meh. Anybody willing to make a decision here? Should I just fix NEWS.md everywhere so it is aligned?

larskarlitski commented:
I prefer it as non-preformatted text, which is why I did that the last rounds. Don't have a strong opinion either way, though.

dvdhrm commented:
> I prefer it as non-preformatted text, which is why I did that the last rounds. Don't have a strong opinion either way, though.

But... NO! I need someone to tell me to go fix all `NEWS.md` to look nice. If nobody tells me to, I have no excuse why I did it!?!?

(btw., I force-pushed a fixed announcement)

ochosi committed:
logo: Make the logo 75% lighter

Previously, the jpg was roughly 4000*1000px, I have scaled this down to
half the width now. 2000px felt ok.
Also, I dropped the quality/compression level from 92 to 90, further
reducing the size. So hopefully this loads fairly instantaneously now.

Fixes #12

ondrejbudai reviewed (approved):
Thanks!
